sample-java-model/src/main/java/com/vimeo/sample_java_model/NativeJavaModel.java

```diff
@@ -5,29 +5,77 @@
 @UseStag
 public class NativeJavaModel {
 
-    String topLevel;
+    private String mTopLevel;
+
+    public String getTopLevel() {
+        return mTopLevel;
+    }
+
+    public void setTopLevel(String mTopLevel) {
+        this.mTopLevel = mTopLevel;
+    }
 
     @UseStag
     public static class Nested {
-        String nested;
+        String mNested;
+
+        public String getNested() {
+            return mNested;
+        }
+
+        public void setNested(String mNested) {
+            this.mNested = mNested;
+        }
     }
 
     public static class NestedWithoutAnnotation {
-        String nestedWithoutAnnotation;
+        String mNestedWithoutAnnotation;
+
+        public String getNestedWithoutAnnotation() {
+            return mNestedWithoutAnnotation;
+        }
+
+        public void setNestedWithoutAnnotation(String mNestedWithoutAnnotation) {
+            this.mNestedWithoutAnnotation = mNestedWithoutAnnotation;
+        }
     }
 
     @UseStag
     public static class NestedExtension extends NativeJavaModel {
-        String nestedExtension;
+        String mNestedExtension;
+
+        public String getNestedExtension() {
+            return mNestedExtension;
+        }
+
+        public void setNestedExtension(String mNestedExtension) {
+            this.mNestedExtension = mNestedExtension;
+        }
```

Comment:
They are not required, but I meant to make all these members private so they should be required. Good catch